docs/index.html

```diff
@@ -7487,6 +7487,7 @@ <h3>
                                         </button>
                                     </td>
                                 </tr>
+                                <!-- /ko -->
```

Comment:
Any files in the `docs` folder should be ignored, as we update these automatically when the website is updated.
```suggestion
```